Source/MaestroDiag.cpp

```diff
@@ -513,17 +513,17 @@ Maestro::DiagFile (const int step,
             diagfile2 << std::setw(setwVal) << std::left << t_in;
             diagfile2 << std::setw(setwVal) << std::left << enuc_max;
 
-            const Real coordenuc2 = (AMREX_SPACEDIM <2) ? 0.0 : coord_enucmax[1];
-            const Real coordenuc3 = (AMREX_SPACEDIM <3) ? 0.0 : coord_enucmax[2];
+            const Real coord_enuc_y = (AMREX_SPACEDIM <2) ? 0.0 : coord_enucmax[1];
```

Comment:
There's still a few of these `AMREX_SPACEDIM <2` that can be removed

Reply:
I am pretty sure all these are done now